mergadoapiclient/client.py

```diff
@@ -25,11 +26,23 @@ def build(credentials, api_version='0.3'):
         NotImplementedError('Grant type not implemented.')
 
 
+def retry_request(make_request, retry_status_list, tries):
+    for i in range(tries):
+        wait = 2 ** i
```

Comment:
tohle by šlo hodit až za ten if - většinou to totiž vůbec nebude potřeba počítat. :slightly_smiling_face: